src/controllers/graphql/resolvers/user/userResolver.js

```diff
@@ -22,7 +22,12 @@ module.exports = {
         : await authenticatedUserInstance.unFollowSteemUser(args.steemUsernameToUnfollow);
     },
     getProfileInformation: async (_, args) => {
-      return await UserService.getUserInfo(args.name);
+      return await UserService.getUserProfileInfo(args.name);
```

Comment:
Also, you can deconstruct `name` from `args`
`getProfileInformation: async (_, { name }) => {
   ...
},
`